sqlitedict.py

```diff
@@ -170,9 +170,12 @@ def __init__(self, filename=None, tablename='unnamed', flag='c',
 
         logger.info("opening Sqlite table %r in %s" % (tablename, filename))
         MAKE_TABLE = 'CREATE TABLE IF NOT EXISTS "%s" (key TEXT PRIMARY KEY, value BLOB)' % self.tablename
-        self.conn = self._new_conn()
-        self.conn.execute(MAKE_TABLE)
-        self.conn.commit()
+        try:
+            self.conn = self._new_conn()
+            self.conn.execute(MAKE_TABLE)
+            self.conn.commit()
+        except sqlite3.OperationalError as e:
+            raise RuntimeError(str(e))
```

Comment:
Why is it better to raise RuntimeError than sqlite3.OperationalError here?

In my opinion, it's not a good idea because it hides useful information from the user.